Dikke Ploaten/Controller/BaseAlbumListTableViewController.swift

```diff
@@ -67,6 +67,10 @@ class BaseAlbumListTableViewController: UITableViewController {
 			
 			cell.updateUI(forAlbum: album)
 		}
+		
+		//Uncomment next line when using "platen" instead of "platenDiscogs"
+		cell.isUserInteractionEnabled = false
```

Comment:
Waarom zijn cellen niet meer clickable?

Reply:
De JSON die ik van Discogs binnenkrijg bevat enkel title, artist en thumb. Description, tracklist e.d. niet waardoor detailscherm van album (so far) niet nuttig is. Mits wat opzoekwerk en meer tijd, zou het wel mogelijk zijn denk ik.

Reply:
Pas dan de comment aan zodat het duidelijk is dat dit door missing info is. Dan zal ik approven